Fixed .NET 4.7.1 install instruction

DevinvN opened the pull request:
Fixed .NET 4.7.1 install instruction, the instruction installed .NET 4.7 instead of .NET 4.7.1

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

cjgronlund commented:
@4c74356b41 
Good good catch! ;>) Thank you for your contribution.
--Carolyn

cjgronlund commented:
@barbkess

DevinvN committed:
Fixed .NET 4.7.1 install instruction

Fixed .NET 4.7.1 install instruction, the instruction installed .NET 4.7 instead of .NET 4.7.1
